### DISCUSSION TITLE
Feeling difficult to code

### DISCUSSION THREAD

**Discussion by PranauvShanmuganathan** [codewars/codewars.com#3183] [General]
Hi Guys ,
Im trying to arrive at an approach , I always fail to optimise the code . This makes me feel like im unfit for IT could anyone suggest a way that I can do better in coding .

**Comment by hobovsky** [comment]
Please mind that Codewars Github is not meant for help with general programming or learning questions. It's meant to discuss matters related to Codewars platform, the website, its functions, etc.

For help with this type of questions as in your post, please visit your favorite general programming forum, a learning community, or you can ask it in `#posts` or `#beginner` sections of Codewars Discord.

I am going to close this discussion as off-topic for the Codewars Github.
